docs/Release_Notes.md

```diff
@@ -1,13 +1,11 @@
-GFS V16.3.20 RELEASE NOTES
+GFS V16.3.21 RELEASE NOTES
 
 -------
 PRELUDE
 -------
-
-The upstream OBSPROC package is updated to v1.3. Along with this are the following companion updates:
-* workflow and UFS_UTILS package updates to use the new AFWA global snow file due to the hemispheric snow files being phased out
-* updated GSI code and convinfo file for saildrone observations
-* Updated GSI convinfo to switch on PlanetIQ GPSRO data
+Update GCIP script and parm file for NESDIS GMGSI product updates:
+- Update GCIP script to work with the new filenames of Global Mosaic of Geostationary Satellite Imagery (GMGSI) products.
+- Update GCIP parm file because one sensor number is new to the current GCIP configuration, and the locations of multiple senors are changed.
```

Comment:
```suggestion
* Update GCIP script to work with the new filenames of Global Mosaic of Geostationary Satellite Imagery (GMGSI) products.
* Update GCIP parm file because one sensor number is new to the current GCIP configuration, and the locations of multiple senors are changed.
```